File: GymApp.Client/src/types/member.ts
export interface Member {
    id: number;
    firstName: string;
    lastName: string;
    email: string;
    phoneNumber: string;
    memberSince: string;
    isActive: boolean;
}

export interface CreateMember {
    firstName: string;
    lastName: string;
    email: string;
    phoneNumber: string;
    dateOfBirth: string;
}

File: GymApp.API/Models/Member.cs
namespace GymApp.API.Models

{
    public class Member
    {
        public int Id { get; set; }
        public string FirstName { get; set; } = string.Empty;
        public string LastName { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public string PhoneNumber { get; set; } = string.Empty;
        public DateTime MemberSince { get; set; } = DateTime.UtcNow;
        public bool IsActive { get; set; } = true;
    }
}